### ISSUE TITLE
Beacon-node crashing on Mainnet

### ISSUE THREAD

**Issue by b-m-f** [sigp/lighthouse#2170]
## Description

This morning my beacon-node crashed with 
```
Jan 22 04:25:06 command lighthouse[634]: thread 'tokio-runtime-worker' panicked at 'assertion failed: now >= lock.elapsed', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-0
Jan 22 04:25:06 command lighthouse[634]: note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

## Version

1.0.6

## Present Behaviour

Upgraded to 1.1.0 already

## Expected Behaviour
Not crash

## Steps to resolve

No idea, also cant reproduce as the validator is live

**Comment by pawanjay176** [comment]
@b-m-f what hardware setup are you using?

**Comment by b-m-f** [comment]
@pawanjay176 
Debian 10
1vCPU
2GB RAM

Will take a look at the Grafana DashBoard later to see if there is helpful information

**Comment by b-m-f** [comment]
So here is a small update:

All except one metric seem to be normal and similar before and after the crash. 
The only thing that stands out is the following:

![image](https://user-images.githubusercontent.com/2843450/105578533-45e91680-5d81-11eb-971a-1a728a3b4fc5.png)

**Comment by paulhauner** [comment]
Looks very similar to this other tokio panic: https://github.com/sigp/lighthouse/issues/2099

**Comment by paulhauner** [comment]
Closing since we believe this is resolved in #2172.
